### ISSUE TITLE
Issue with GOOSE

### ISSUE THREAD

**Issue by mcourtot** [geneontology/go-site#270]
From helpdesk: http://jira.geneontology.org/browse/GO-1410

Same request sometimes goes through sometimes not; it seems once there is a fail (e.g. by using the nucleus sample query from the drop down menu) GOOSE is stuck in failure mode.

Also I couldn't see what was wrong with the sample nucleus query causing the fail. I thought maybe the limit of results returned, but even setting this as unlimited didn't help.

The following query works for me:
SELECT DISTINCT
descendant.acc,
descendant.name,
descendant.term_type
FROM
term INNER JOIN
graph_path ON (term.id=graph_path.term1_id) INNER JOIN
term AS descendant ON (descendant.id=graph_path.term2_id)
WHERE
term.acc='GO:0006260' AND
distance <> 0 ;
where GO:0006260 is the ID for DNA replication.

That being said, there is something odd going on - the example query for nucleus form the wiki doesn't work for me either. If I open a new window and paste in the above, replacing the GO ID of DNA replication with the GO ID of nucleus, GO:0005634, the request goes through. If I then use the 'nucleus' label as in the original query, the request goes through - it's almost as if once the query fail the server keeps the failed state throughout.

**Comment by kltm** [comment]
TL;DR: one of the mirrors is broken, but no so broken that GOOSE can detect it.

Dollars to donuts, this is what's going on:  once a mirror has been selected, GOOSE will attempt to stay sticky to it. My thinking is that whatever mirror GOOSE is selecting, the tables have been incompletely or improperly loaded, causing the error. This could even be because the mirror is being reloaded right now.
There are certain circumstances, where is a mirror is _completely_ unresponsive, that GOOSE will attempt to switch, which could cause some of the seemingly random behavior.

GOOSE is not, however, really set to pickup on all of these edges cases--if a mirror is up it is assumed to work. If there is interest, we would have to write a MySQL mirror health client to do various data checks against the mirrors and announce when they did not pass. Given the fact that GOOSE is rather on it's way out, I think effort would probably better spent on the graph client (although I'm happy to be told otherwise, @cmungall ).
